src/dialog/overflow-menu/overflow-menu-pane.component.ts

```diff
@@ -32,5 +34,57 @@ export class OverflowMenuPane extends Dialog {
 		 * so we need to add some compensation)
 		 */
 		this.addGap["bottom"] = pos => position.addOffset(pos, -20, 60);
+
+		setTimeout(() => this.listItems()[0].focus());
+	}
+
+	@HostListener("keydown", ["$event"])
+	hostkeys(event: KeyboardEvent) {
+		this.escapeClose(event);
+		const listItems = this.listItems();
+
+		switch (event.key) {
+			case "ArrowDown":
+				event.preventDefault();
+				if (!isFocusInLastItem(event, listItems))  {
+					const index = listItems.findIndex(item => item === event.target);
+					listItems[index + 1].focus();
+				} else {
+					listItems[0].focus();
+				}
+				break;
+
+			case "Up": // IE specific value
+			case "ArrowUp":
+				event.preventDefault();
+				if (!isFocusInFirstItem(event, listItems))  {
+					const index = listItems.findIndex(item => item === event.target);
+					listItems[index - 1].focus();
+				} else {
+					listItems[listItems.length - 1].focus();
+				}
+				break;
+
+			case "Home":
+				event.preventDefault();
+				listItems[0].focus();
+				break;
+
+			case "End":
+				event.preventDefault();
+				listItems[listItems.length - 1].focus();
+				break;
+
+			case "Enter":
+			case " ":
+				this.doClose();
+				break;
+		}
+	}
+
+	private listItems() {
+		const buttonClass = ".bx--overflow-menu-options__btn";
+		const disabledClass = ".bx--overflow-menu-options__option--disabled";
+		return Array.prototype.slice.call(this.dialog.nativeElement.querySelectorAll(`${buttonClass}:not(${disabledClass})`));
```

Comment:
the `Array.prototype.slice.call` can be replaced with `Array.from`

Reply:
I can't do that because `Array.from` will return an object, and I need an array

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from `Array.from` returns an array